src/ToDo.UI/Views/SettingsPage.xaml

```diff
@@ -41,6 +41,7 @@
 						   Text="{Binding CurrentUser.Email}"
 						   Style="{StaticResource MaterialBody2}" />
 				<Button Content="sign out"
+						x:Uid="SignOutButton"
```

Comment:
Can we update the x:Uid of this page to match the other please @Xiaoy312 ? (x:Uid="**NameOfThePage**_Name")
@carldebilly I don't think the generated code for the plugin already follow the same pattern as the x:Uid in this PR no ? Should we open a figma issue for it ?

Reply:
This PR just cover some of the pages. I havent gone through SettingsPage yet.
The plan is to merge this ASAP, so I dont have to deal with further merging conflicts...

Reply:
And, then I will make the rest in another PR or PRs

Reply:
Ok no problem and I understand ;) go ahead and @carldebilly feel free to give us your opinion regarding the generated code.

Reply:
@agneszitte-nventive you're right it should follow the generated format. We'll need to make sure the names are correct in Figma, but for now, that's correct.